tensorflow/lite/experimental/examples/unity/TensorFlowLitePlugin/Assets/TensorFlowLite/SDK/Scripts/Interpreter.cs

```diff
@@ -152,7 +173,67 @@ public class Interpreter : IDisposable
         TfLiteTensor tensor,
         IntPtr output_data,
         int output_data_size);
+    
+    enum TfLiteGpuInferenceUsage : Int32 {
```

Comment:
Nice! Thanks for pushing forward on this, really cool that this "just works"!

Could we add the GpuDelegate in its own class/file (and make a Delegate interface in a separeate file)? We can pattern it after what we've done in Java, where Delegate.java has a simple interface to return the native handle to the native delegate instance. We can then pass in the Delegate class as an argument to the Interpreter constructor.

Reply:
Sure, I'll get on that and test it again

Reply:
The name delegate is already used in C#. Does TFDelegate work here?

Reply:
Let's call it InterpreterDelegate for now (for the interface), and GpuDelegate (for the implementation).